concepticondata/conceptlists/Araujo-1996-289.tsv

```diff
@@ -0,0 +1,290 @@
+ID	NUMBER	GLOSS	CONCEPTICON_ID	CONCEPTICON_GLOSS	GERMAN	PORTUGUESE
+Araujo-1996-289-1	1	Head	1256.0	HEAD	Kopf	cabeça
+Araujo-1996-289-2	2	Face	1560.0	FACE	Gesicht	rosto
+Araujo-1996-289-3	3	Eye	1248.0	EYE	Auge	olho
+Araujo-1996-289-4	4	Eyebrow	1173.0	EYEBROW	Brauen	sobrancelha
+Araujo-1996-289-5	5	Eyelash	1540.0	EYELASH	Wimpern	pestana
+Araujo-1996-289-6	6	Nose	1221.0	NOSE	Nase	nariz
+Araujo-1996-289-7	7	Ear	1247.0	EAR	Ohr	orelha
+Araujo-1996-289-8	8	Ear Lobe Piercing			Loch im Ohrläppchen	fura do lóbulo da orelha
+Araujo-1996-289-9	9	Mouth	674.0	MOUTH	Mund	boca
+Araujo-1996-289-10	10	Lower Lip			Unterlippe	lábio inferior
+Araujo-1996-289-11	11	Lower Lip Piercing			Loch in der Unterlippe	buraco no lábio inferior
+Araujo-1996-289-12	12	Tooth	1380.0	TOOTH	Zahn	dente
+Araujo-1996-289-13	13	Tongue	1205.0	TONGUE	Zunge	lí­ngua
+Araujo-1996-289-14	14	Moustache	1165.0	MOUSTACHE	Schnurrbart	bigode
+Araujo-1996-289-15	15	Hair	2648.0	HAIR (HEAD)	Haar	cabelo
+Araujo-1996-289-16	16	Straight Hair			schlichtes Haar	cabelo liso
+Araujo-1996-289-17	17	Curly Hair			Wollhaar	cabelo crespo
```

Comment:
I'll resolve these suggestions directly because these German glosses are not mine, but thank you for providing your translation, that way we ensure that the English gloss is accurate.